experimental/traffic-portal/src/app/api/testing/server.service.ts

```diff
@@ -214,7 +222,14 @@ export class ServerService {
 			}
 			return status;
 		}
-		return this.statuses;
+		return this.statuses.map(
+			p => ({
+				description: p.description,
+				id: p.id,
+				lastUpdated: p.lastUpdated,
+				name: p.name
+			})
+		);
```

Comment:
This mapping is not operating on each status in any way, returning an object reconstructed in exactly the same way as the original, so: why?